### ISSUE TITLE
Cannot press Enter in Settings Editor

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#52973]
* Open Settings on Windows
* Try to press <kbd>Enter</kbd>:
* It does not work:
![image](https://user-images.githubusercontent.com/5047891/41917612-1edab7e0-795b-11e8-8ec2-b91b23c56b6e.png)

**Comment by alexandrudima** [comment]
Adding the following to my `keybindings.json` has unblocked me:

```
{
	"key": "enter",
	"command": "-settings.action.editFocusedSetting",
	"when": "inSettingsEditor"
}
```

**Comment by sulkhanp** [comment]
@alexandrudima dupe of #52942 ?

**Comment by roblourens** [comment]
Duplicate of #52973
